src/main/java/mil/dds/anet/resources/PersonResource.java

```diff
@@ -290,4 +291,17 @@ private String validateEmailErrorMessage() {
         : String.format("%s at %s", messageBody, supportEmailAddr);
     return errorMessage;
   }
+
+  @GraphQLQuery(name = "thumbnail")
```

Comment:
Do we need a new graphql method for that? I was suggesting to have the avatar method take an optional parameter that would trigger a scaledown

Reply:
If we use this graphql query, it will only works for a single object. If we use `avatar`, then we can ask for a list of people and get their avatars in a single pass

Reply:
I changed it. Also removed the Thumbail-component as it was no more necessary.